core/vm/logger.go

```diff
@@ -76,6 +77,7 @@ type StructLogger struct {
 	cfg LogConfig
 
 	logs          []StructLog
+	sizeLogs      int
```

Comment:
Ah, possibly, maybe, is this `[]StructLog` thing an array with a counter (somehow I thought this might be a list)?  Then yes I will change this.